cli/src/template_builder.rs

```diff
@@ -436,6 +436,12 @@ fn build_timestamp_method<'a, L: TemplateLanguage<'a>>(
                 time_util::format_absolute_timestamp_with(&timestamp, &format)
             }))
         }
+        "utc" => {
+            template_parser::expect_no_arguments(function)?;
+            language.wrap_string(TemplateFunction::new(self_property, |timestamp| {
+                time_util::format_to_utc(&timestamp)
+             }))
+        }
```

Comment:
Suppose we'll change `.utc()` to return `Timestamp`, what we need to do is to reset `tz_offset` to `0`.